app/presenters/summary/sections/case_details.rb

```diff
@@ -10,6 +10,7 @@ def answers
         [
           Components::ValueAnswer.new(
             :case_type, kase.case_type,
+            show: crime_application.means_passport.exclude?('on_not_means_tested'),
```

Comment:
Provider won't have been asked case_type if the application is not means tested, so we don't want to show this row.

Reply:
Do as a helper method

Reply:
Since the show method now works properly, this isn't needed - as `show` resolves to false if the value is `nil`